modules/client-configuration/pages/client-proxy.adoc

```diff
@@ -21,4 +22,69 @@ You can see a list of clients that are connected to a proxy by clicking the name
 
 A list of chained proxies for a Salt client can be seen by clicking the name of the client in menu:Systems[All], then select the [guimenu]``Connection`` subtab of the [guimenu]``Details`` tab.
 
-If you decide to move any of your clients between proxies or the server you need to repeat the registration process from the beginning.
+
+
+== Move Clients between Proxies
+
+You can move Salt and Salt SSH push clients between proxies without the need to repeat the registration process.
+
+
+
+[NOTE]
+====
+If you want to move a traditional client between proxies or any client between chained proxies you must repeat the registration process from the beginning.
+====
+
+// Procede as follows:
+
+
+
+.Procedure: Moving Salt or  Salt SSH Push Client between Proxies
+
+. In the {productname} {webui}, navigate to the [guimenu]``System Details`` page for the client you want to move between proxies.
+. Navigate to the [guimenu]``Connection`` tab.
+. From the [guimenu]``New Proxy`` drop-down menu select the proxy you want the client to move to, and click btn:[Change Proxy].
+
+
+
+.Procedure: Moving Multiple Salt or Salt SSH Push Clients between Proxies with SSM
+
+. In the {productname} {webui}, navigate to menu:Systems[System List] and check each client to move, this adds the clients to the system set manager.
+. Navigate to menu:Systems[System Set Manager], and go to the menu:Misc[Proxy]`` tab.
+. From the [guimenu]``New Proxy`` drop-down menu select the proxy you want the clients to move to, and click btn:[Change Proxy].
+
+
+
+The same functionality is also available with the [systemitem]``system.changeProxy`` API call.
+
+
+
+=== Background Information
+The effect of this function differs between normal Salt clients and Salt SSH push clients.
+
+
+==== Normal Salt Clients
+
+The function schedules a Salt state action, which modifies [literal]``master:`` setting in the [path]``susemanager.conf`` Salt client configuration file to point to the new proxy.
+Then the function restarts the Salt client.
+
+[NOTE]
+====
+Changing [literal]``master:`` by manually editing the [path]``susemanager.conf`` file has the same effect and is supported, too.
+====
+
+When the minion restarts and reconnects via the new proxy, the server updates the proxy path in the database and schedules another action for refreshing the channel URLs.
+
+
+
+==== Salt SSH Push Clients
+
+The function updates the proxy path immediately in the database and new action
+for refreshing the channel URLs is scheduled.
+
+
+
+== Move Clients from Proxies to the Server
+
+If you want to move any Salt or traditional client from proxies to the server you must repeat the registration process from the beginning.
+
```

Comment:
Moving salt client from proxy to server is supported, just select "None" from proxy list.

Reply:
ok